crates/handlers/src/graphql/mod.rs

```diff
@@ -44,7 +46,22 @@ use rand::{thread_rng, SeedableRng};
 use rand_chacha::ChaChaRng;
 use sqlx::PgPool;
 use tracing::{info_span, Instrument};
+use ulid::Ulid;
+
+mod model;
+mod mutations;
+mod query;
+mod state;
 
+#[cfg(test)]
+pub use state::BoxState;
```

Comment:
I don't understand why is this under `cfg(test)`?

Reply:
It got a warning for being unused when not in tests :)

Reply:
using it below directly serves as a workaround, so don't need this anymore